docs/mecsimcalc-library/plotdraw.md

```diff
@@ -173,10 +35,11 @@ Creates a blank image with specified width and height, displaying a grid.
 
 #### Arguments:
 
-| Argument     | Type    | Description                                          |
-| ------------ | ------- | ---------------------------------------------------- |
-| **`width`**  | **int** | The width of the image in pixels. (Default is 1000)  |
-| **`height`** | **int** | The height of the image in pixels. (Default is 1000) |
+| Argument     | Type               | Description                                             |
+| ------------ | ------------------ | ------------------------------------------------------- |
+| **`width`**  | **int** (optional) | The width of the image in pixels. (Default is 800)      |
```

Comment:
The format you're using for default is different from the other page. In the other one you had 

> Default is `800`

Reply:
I think the brackets look nicer but having the value in backticks is good as well

Reply:
Thanks for pointing it out, but I'll do it another time